draft-ietf-quic-invariants.md

```diff
@@ -234,7 +281,7 @@ and is not constrained by this specification.
 The remainder of the packet has version-specific semantics.
 
 
-## Connection ID
+## Connection ID {#connection-id}
```

Comment:
Nothing wrong with doing this, but it's pointless since this would be the default anchor generated for this section title anyway.